recipes/gatsby-cli/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "5.13.3" %}
+
+package:
+  name: gatsby-cli
+  version: {{ version }}
+
+source:
+  url: https://registry.npmjs.org/gatsby-cli/-/gatsby-cli-{{ version }}.tgz
+  sha256: 22419fe3354ce4a4e373aaa54160294b8d5cc5ab95ad6b632b07a047c6287378
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - findutils
+    - gcc  # [linux]
```

Comment:
You should never need to depend on `gcc` in recipes. If really required, add an explanative comment.
```suggestion
```

Reply:
The issue here is that the the `msgpackr-extract`, a dependency of `gatsby-cli`, hardcodes `gcc` as the compiler executable name rather than using CC on Linux: https://github.com/kriszyp/msgpackr-extract/blob/db8f5b97e7aaab0bcad95f10582915d417153689/binding.gyp#L3.  

Would it be better in this scenario to use symlink from `CC` to `gcc` in `$BUILD_PREFIX/bin`?  I'm not sure what our established best practice is for this problem.